policy/pipeline/artifacthub-pkg.yml

```diff
@@ -0,0 +1,19 @@
+# from https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.0.1
+name: OPA Pipeline Policies
+displayName: OPA pipeline Policies
+createdAt: 2023-01-05T21:27:55.549418+00:00
+description: OPA policies enforcing standards on a tekton pipeline definition
+readme: |
+  OPA policies enforcing standards on a tekton pipeline definition
+install: |
+  `conftest pull git::https://github.com/hacbs-contract/ec-policies.git//policy`
+  Configure your org-specific `--data` as necessary from [data/](https://github.com/hacbs-contract/ec-policies/tree/main/data)
```

Comment:
Do we have any org specific data for pipeline validation?

Reply:
I guess i don't see any. I'll remove.